### ISSUE TITLE
upgrade to 0.34 broke queries with no results

### ISSUE THREAD

**Issue by 2-5** [vuex-orm/vuex-orm#504]
This code worked in 0.33, but fails in 0.34, because `first()` returns `undefined` instead of `null` when the query has no results:
```typescript
const setting: Setting | null = Setting.query()
  .where('name', name)
  .first()

if (setting === null) {
  return defaultValue
}

// TypeError: Cannot read property 'value' of undefined
const value: string = setting.value
```

**Comment by kiaking** [comment]
Thank you for the report! Confirmed. Fix is on the way.

**Comment by kiaking** [comment]
This is now fixed at Vuex ORM 0.34.1! Thank you for reporting it!
https://github.com/vuex-orm/vuex-orm/releases/tag/v0.34.1
